http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -598,6 +606,45 @@ public void setNotFoundOnMissingBody(boolean notFoundOnMissingBody) {
         this.notFoundOnMissingBody = notFoundOnMissingBody;
     }
 
+    /**
+     * Returns whether the semicolon is considered a normal character in the query.
+     * A "normal" semicolon is one that is not used as a parameter separator.
+     *
+     * @return {@code true} if the semicolon is a normal character, {@code false} otherwise.
+     * @since 4.8
+     */
+    public boolean isSemicolonIsNormalChar() {
```

Comment:
I'm not sure this belongs in `HttpServerConfiguration` since I am unsure the other server implementations (jetty, tomcat etc.) allow this kind of configuration. For the moment it should be moved to `NettyHttpServerConfiguration`. @yawkat WDYT?

Reply:
Hm actually since this also affects our non-netty copy of QueryStringDecoder